packages/app/src/App.vue

```diff
@@ -1,6 +1,8 @@
 <template>
-  <div class="app flex">
-    <router-view />
+  <div class="app bg-green-800 flex">
```

Comment:
I would suggest another color as background based on
https://color.adobe.com/de/create/color-wheel

maybe use ```bg-indigo-800``` ```bg-blue-900``` or ```bg-blue-800```

Reply:
I checked your suggestions, but currently I would stick to the color we have.